### PR TITLE
Update language.php

### PR DESCRIPTION
**Pull request by Headhunterzpw**

### CODE REVIEW

File: styles/language/Dutch/language.php [hunk 1]
```diff
@@ -688,7 +688,7 @@
     // Other
     'page_loaded_in' => 'Pagina geladen in {x} seconden', // Don't replace {x}; 's' stands for 'seconds'
     'none' => 'Geen',
-    '404' => 'Sorry, we konden deze pagina niet vinden. Weet je zeker dat je de URL correct heeft gespeld?'
+    '404' => 'Sorry, we kunnen de pagina niet vinden die je probeert te bezoeken. Controleer de URL en probeer het opnieuw, als het nog niet lukt neem dan contact op met de Administrator'
```
Comment (rodymol123) [inline styles/language/Dutch/language.php]: Dit is niet helemaal grammaticaal correct ;)

Comment (TheSander562) [inline styles/language/Dutch/language.php]: Jawel toch, wat is er dan fout?

Comment (Awtra) [inline styles/language/Dutch/language.php]: Just want to make sure if this translation is correct. Any updates on that? @TheSander562 @rodymol123

Comment (TheSander562) [inline styles/language/Dutch/language.php]: yes, this is correctly updated.

**Comment by samerton** [comment]
Thanks

**Comment by badbojus** [comment]
what language it is :O ?

**Comment by TheSander562** [comment]
It's called Dutch, everyone and i speak it in the Netherlands.

**Comment by badbojus** [comment]
ccol, thanks! :)

**Comment by TheSander562** [comment]
youre welcome

**Pull request merged**